service/eventer/github/client.go

```diff
@@ -25,6 +24,11 @@ const (
 	// etagHeader is the header used for etag.
 	// See: https://en.wikipedia.org/wiki/HTTP_ETag.
 	etagHeader = "Etag"
+
+	// LatestDeploymentUrlFormat is the format string used to compute the Github
+	// API URL used to fetch the latest deployment event for a specific
+	// environment.
+	LatestDeploymentUrlFormat = "https://api.github.com/repos/%s/%s/deployments?environment=%s"
```

Comment:
when initializing the TPO the very first time we fill it with the latest deployment event information we can find. This eases the bootstrapping of host clusters a bit which is one of the reasons why we are doing all this jazz.